andela/ah-jarvis-frontend
# #159961942 Setup Airbnb Eslint

#### What does this PR do?
Enables the use of Airbnb eslint which helps in following the best syntax for React/Redux/JXS

#### How should this be manually tested
- For vscode users:-
`install eslint extension by Dirk Baeumer`

- clone this project
`git clone https://github.com/andela/ah-jarvis-frontend.git`

- Run the following commands
```
cd ah-jarvis-frontend

git checkout ch-airbnb-eslint-159961942
```
- You will notice red lines in areas where the syntax is incorrect

#### Screenshots
None

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#159961942](https://www.pivotaltracker.com/story/show/159961942) Setup an empty react project
